### DISCUSSION TITLE
Cell Review

### DISCUSSION THREAD

**Discussion by richard1015** [jdf2e/nutui-react#22] [General]
- 功能、基本交互测试
   - [ ] 单行cell组件 底部有下划线 .nut-cell:after
   - [ ] 自定义左侧 Icon 区域
- 文档完整性验证
   - [ ] import { Cell,CellGroup } from '@nutui/nutui-react';
   - [ ] ...
- props 所有属性测试
   - [ ] ...
   - [ ] ...
- props 异步修改组件是否动态更新
   - [ ] ...
   - [ ] ...
- emits 所有事件测试
   - [ ] ...
   - [ ] ...
- npm run build 子组件是否被构建
   - [ ] export CellGroup 未导出
   - [ ] ...
- 其它
   - [ ] 全局 reset.css 需删除
   - [ ] ...

**Comment by dushoujun** [comment]
1. 设置className会把原本的替换掉，导致没有样式了
2. switch组件引入未完善
3. descTextAlign设置未生效
